readme.md

```diff
@@ -87,31 +87,62 @@ Large files are used during load testing. Large test files are created using a c
  [2019 Large File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml).
```

Comment:
Can you drop these two files down into a bulleted list?

Reply:
Yep!